task/upload.go

```diff
@@ -300,20 +343,24 @@ func complementCatalystPipeline(tctx *TaskContext, assetSpec api.AssetSpec, call
 	}
 
 	if !FlagCatalystCopiesSourceFile {
-		input, err := readLocalFile(0.95)
-		if err != nil {
-			return nil, err
-		}
 		fullPath := videoFileName(playbackID)
-		fileUrl, err := osSess.SaveData(tctx, fullPath, input, nil, fileUploadTimeout)
-		if err != nil {
-			return nil, fmt.Errorf("error uploading file=%q to object store: %w", fullPath, err)
-		}
 		assetSpec.Files = append(assetSpec.Files, api.AssetFile{
 			Type: "source_file",
 			Path: toAssetRelativePath(playbackID, fullPath),
 		})
-		glog.Infof("Saved file=%s to url=%s", fullPath, fileUrl)
+
+		// in case of encrypted input, file will have been copied in the beginning
+		if params.Encryption.Key == "" {
```

Comment:
This is failing in all tests as params.Encryption is undefined, wrong version of lapi?

Reply:
Yeah, probably was only working after merging with your `dev`